odk1-src/central-install-digital-ocean.rst

```diff
@@ -0,0 +1,240 @@
+
+Once you have one account, you do not have to go through this process again for future accounts: you can log into the website with your new account, and directly create new users that way.
+
+You're done! Congratulations. In the future, you may wish to consult the :doc:`central-upgrade` guide, but for now you may begin using ODK Central. The :doc:`central-using` sections can help you with your next steps if you aren't sure how to proceed.
+
+.. _central-install-digital-ocean-advanced:
+
+Advanced Configuration Options
+==============================
+
+The following sections each detail a particular customization you can make to your server setup. Most installations should not need to perform these tasks, and some of them assume some advanced working knowledge on administering Linux web servers. If you aren't sure what something means, the best option is probably to skip the section completely.
+
+.. _central-install-digital-ocean-dkim:
+
+Configuring DKIM
+----------------
+
+DKIM is a security trust protocol which is used to help verify mail server identities. Without it, your sent mail is likely to be flagged as spam. If you intend to use a custom mail server (see the following section), these instructions will not be relevant to you. Otherwise:
+
+1. Ensure that your server's name in DigitalOcean `matches your full domain name <https://www.digitalocean.com/community/questions/how-do-i-setup-a-ptr-record?comment=30810>`_, and that the `hostname does as well <https://askubuntu.com/questions/938786/how-to-permanently-change-host-name/938791#938791>`_. If you had to make changes for this step, restart the server to ensure they take effect.
+2. Now, you'll need to generate a cryptographic keypair. Run these commands:
+
+   .. code-block:: console
+
+     mkdir ~/dkim
+     cd ~/dkim
+     openssl genrsa -out rsa.private 1024 
+     openssl rsa -in rsa.private -out rsa.public -pubout -outform PEM
+
+3. With the contents of the public key (``cat dkim/rsa.public``), you'll want to create two new TXT DNS records:
+
+   1. At the location ``dkim._domainkey.YOUR-DOMAIN-NAME-HERE``, create a new ``TXT`` record with the contents ``k=rsa; p=PUBLIC-KEY-HERE``. You only want the messy text *between* the dashed boundaries, and you'll want to be sure to remove any line breaks in the public key text, so that it's all only letters, numbers, ``+``, and ``/``.
+   2. At your domain name location, create a new ``TXT`` record with the contents ``v=spf1 a mx ip4:SERVER-IP-ADDRESS-HERE -all`` where you can obtain the server IP address from the DigitalOcean control panel.
+
+4. Finally, you'll need to configure EXIM to use the cryptographic keys you generated. First, navigate to your ``central`` project directory (``cd ~/central``).
+
+   1. ``mkdir files/dkim`` to create a new folder, then ``cd files/dkim`` to go to that folder.
```

Comment:
i moved the `cd` line into the numbered sublist and reworded to make this distinction clearer.

Reply:
okay never mind i actually completely redid this section upon remembering/rediscovering that docker will ignore files it cannot find. it's way simpler now. i mostly tried it but if you have time to give it another quick go it would be good to get a from-scratch run on it.

Reply:
Didn't get a chance to try, but the change is much better.